patches/tahoma2d-flatpak.patch

```diff
@@ -0,0 +1,89 @@
+From ffdd8433509c92fc634bb1eeeb058b0f15363966 Mon Sep 17 00:00:00 2001
+From: manongjohn <19245851+manongjohn@users.noreply.github.com>
+Date: Sun, 24 Dec 2023 10:34:10 -0500
+Subject: [PATCH 1/2] Add Flatpak bin directory for ffmpeg & rhubarb detection
+
+---
+ toonz/sources/toonzlib/thirdparty.cpp | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/toonz/sources/toonzlib/thirdparty.cpp b/toonz/sources/toonzlib/thirdparty.cpp
+index 36691317d..30a3f2ef6 100644
+--- a/toonz/sources/toonzlib/thirdparty.cpp
++++ b/toonz/sources/toonzlib/thirdparty.cpp
+@@ -110,6 +110,7 @@ QString autodetectFFmpeg() {
+ #endif
+ 
+ #ifndef _WIN32
++  folderList.append("/app/bin");
```

Comment:
a proper fix upstream would be to resolve `PATH` environment.

Reply:
OT build does this, however, I already have a change to T2D code for this but wont be available until the next release or fix-release if i have one. In the meantime, there are 2 temp patches being applied until the next release. The patches are necessary for the current T2D version in order to run correctly from flatpak

Reply:
I know about this patches: I made them.

Reply:
the one for opentoonz I mean

Reply:
Yes, but that patch alone doesnt work. I understand why it was added and it will be fixed properly in the next release. Until then, i request these T2D specific patches remain. I have a note in the manifest to remove it when i update to the next release.

Reply:
it's fine to have them, I just wanted to nudge as it not being the right solution upstream since you are upstream.

Reply:
As soon as I saw the patch, I new immediately I was going to correct the T2D code (which I did), so I agree with you there.  Additionally, the OT patch you created can't apply to T2D code anyway so I just created a short term patch knowing the next release will correct the issue.